sdk/txbuild/sequence.go

```diff
@@ -5,3 +5,17 @@ const m = 2
 func startSequenceOfIteration(startSequence int64, iterationNumber int64) int64 {
 	return startSequence + iterationNumber*m
 }
+
+type TransactionType string
+
+const (
+	TransactionTypeDeclaration TransactionType = "declaration"
+	TransactionTypeClose       TransactionType = "close"
+)
+
+func SequenceNumberToTransactionType(startingSeqNum, seqNum int64) TransactionType {
+	if startingSeqNum%m == seqNum%m {
+		return TransactionTypeDeclaration
+	}
+	return TransactionTypeClose
+}
```

Comment:
Could you add some tests that demonstrate even and odd starting sequences, and then both tx types? 😄

Reply:
that's a good idea https://github.com/stellar/experimental-payment-channels/pull/324/commits/c1426f119d29d722dc892b2e157685af61b5937d

writing the test made me think that we should add a `TransactionTypeOpen` (if they're the same)

also there's 1 unique case of the startingSequence + 1, which isn't a transaction type. Maybe we should add a type for that as well (it would be something like TransactionTypeNone)

Reply:
added them here
https://github.com/stellar/experimental-payment-channels/pull/324/commits/d5e1329d027b68d5ba9745db8ebd74415bf77f2a

Reply:
I wonder if we can focus on the transaction types being for close agreements then, because I don't think we get much bang out of adding other types.

I wouldn't worry about adding the none case either, we don't sign txs with those sequence numbers. I'd just panic in that case, it's code that should never be hit.

Reply:
added the panic https://github.com/stellar/experimental-payment-channels/pull/324/commits/0b1d1df7aaa8b51a4eef16e4d891caf4c5612fd8

imo I think the `TransactionTypeFormation` makes sense, isn't used now but may save a bug down the line if someone means to get the formation tx seq and not the declaration tx seq 🤷 

up to you though I can remove if it's added code for no reason